components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -9811,6 +9811,18 @@ public String getEarliestRevisionUUID(String apiUUID) throws APIManagementExcept
         return apiMgtDAO.getEarliestRevision(apiUUID);
     }
 
+    /**
+     * Get the latest revision UUID from the revision list for a given API
+     *
+     * @param apiUUID API UUID
+     * @return Earliest revision's UUID
```

Comment:
Shall we change the as Latest revision UUID since the method name is stated as that?